sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -322,6 +376,74 @@ def body_as_json(self, encoding="UTF-8"):
         except Exception as e:
             raise TypeError("Event data is not compatible with JSON type: {}".format(e))
 
+    @property
+    def content_type(self):
+        # type: () -> Optional[str]
+        """The content type descriptor.
+        Optionally describes the payload of the message, with a descriptor following the format of RFC2045, Section 5,
+        for example "application/json".
+        :rtype: str
+        """
+        if not self._raw_amqp_message.properties:
+            return None
+        try:
+            return self._raw_amqp_message.properties.content_type.decode("UTF-8")
+        except (AttributeError, UnicodeDecodeError):
+            return self._raw_amqp_message.properties.content_type
+
+    @content_type.setter
+    def content_type(self, value):
+        # type: (str) -> None
+        if not self._raw_amqp_message.properties:
+            self._raw_amqp_message.properties = AmqpMessageProperties()
+        self._raw_amqp_message.properties.content_type = value
+
+    @property
+    def correlation_id(self):
+        # type: () -> Optional[str]
+        # pylint: disable=line-too-long
```

Comment:
nit: is the pylint needed here?
if it's for suppressing the doc, than we could update the comment to start from a new line

Reply:
no just forgot to remove it, good catch